META6.json

```diff
@@ -76,6 +76,7 @@
     "Bailador::Sessions" : "lib/Bailador/Sessions.pm",
     "Bailador::Sessions::Store" : "lib/Bailador/Sessions/Store.pm",
     "Bailador::Sessions::Store::Memory" : "lib/Bailador/Sessions/Store/Memory.pm",
+    "Bailador::Status" : "lib/Bailador/Status.pm",
```

Comment:
would you please move this file to Bailador/Feature/. I think that would be a good place for all "non core"/plugin-like features. ( Bailador/Gradual.pm might be better placed in Bailador/Feature as well. )